packages/next/client/image.tsx

```diff
@@ -680,7 +714,46 @@ export default function Image({
     handleLoading(imgRef, srcString, layout, placeholder, onLoadingCompleteRef)
   }, [srcString, layout, placeholder, isVisible])
 
-  return (
+  return isRaw ? (
```

Comment:
Curious why we need to create the `isRaw` variable. Can we not check if `layout === 'raw'` here like we have elsewhere?